jigglypuffRL/deeprl/ddpg/ddpg.py

```diff
@@ -164,7 +159,7 @@ def select_action(self, state, deterministic=True):
         with torch.no_grad():
             action = self.ac.get_action(
                 torch.as_tensor(state, dtype=torch.float32, device=self.device),
-                deterministic=deterministic,
```

Comment:
I'd changed this here. For DDPG and TD3 since they take deterministic actions always.

Reply:
Okay. Sounds good